code/game/jobs/job_exp.dm

```diff
@@ -0,0 +1,207 @@
+
+// Defines
+
+#define EXP_TYPE_CREW			"Crew"
+#define EXP_TYPE_COMMAND		"Command"
+#define EXP_TYPE_ENGINEERING	"Engineering"
+#define EXP_TYPE_SECURITY		"Security"
+#define EXP_TYPE_SILICON		"Silicon"
+#define EXP_TYPE_SERVICE		"Service"
+#define EXP_TYPE_MEDICAL		"Medical"
+#define EXP_TYPE_SCIENCE		"Science"
+#define EXP_TYPE_SUPPLY			"Supply"
+#define EXP_TYPE_SPECIAL		"Special"
+#define EXP_TYPE_GHOST			"Ghost"
+#define EXP_TYPE_EXEMPT			"Exempt"
+
+// Globals, Defines, Etc
+
+
+
+
+// Admin Verbs
+
+
+/client/proc/cmd_admin_check_player_exp()	//Allows admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player Playtime"
+	if(!check_rights(R_ADMIN))
+		return
+	var/msg = "<html><head><title>Playtime Report</title></head><body>Playtime:<BR><UL>"
+	for(var/client/C in clients)
+		var/list/play_records = params2list(C.prefs.exp)
+		var/gen_exp_time = text2num(play_records[EXP_TYPE_CREW])
+		gen_exp_time = num2text(round(gen_exp_time / 60)) + "h"
+		msg += "<LI> - [key_name_admin(C.mob)]: <A href='?_src_=holder;getplaytimewindow=\ref[C.mob]'>" + C.get_exp_general() + "</a></LI>"
```

Comment:
... this topic uses a reference to the client's mob, which gets dereferenced back to the mob in the topic code, and then that mob's client gets used.

Just use a client reference here.